admin/settings/class-general.php

```diff
@@ -72,17 +72,17 @@ public function set_form_fields()
                 'type'  => 'checkbox',
                 'name'  => 'add_v4v_rss_tag',
                 'value' => 'on',
-                'label' => __('Enable Value4Value tag', 'lnp-alby'),
-                'description' => __('Add the podcast:value tag to your RSS feed. Configure the node key (and custom key/value if needed)', 'lnp-alby'),
+                'label' => __('Enable Value 4 Value tag', 'lnp-alby'),
+                'description' => __('Add the podcast:value tag to your RSS feed. Configure the node address (and custom key/value if needed).', 'lnp-alby'),
             ),
         );
 
         $fields[] = array(
             'tab'     => 'value4value',
             'field'   => array(
                 'name'  => 'v4v_node_key',
-                'label' => __('Node key', 'lnp-alby'),
-                'description' => __('Node address - the lightning node public key', 'lnp-alby'),
+                'label' => __('Node Address', 'lnp-alby'),
```

Comment:
it's actually the public key, isn't it?

Reply:
Yes, I think "public key" is the very correct answer. Some websites like amboss.space use "address"
I don´t care what we use. I thought to make it more consistent and easy to understand for the user.